docs/docusaurus/docs/core/introduction/try_gx.py

```diff
@@ -0,0 +1,58 @@
+"""
+This example script allows the user to try out GX by validating Expectations
+ against provided sample data.
+
+The <snippet> tags are used to insert the corresponding code into the
+ Great Expectations documentation.  They can be disregarded by anyone
+ reviewing this script.
+"""
+
+# <snippet name="docs/docusaurus/docs/core/introduction/try_gx.py full example script">
+# <snippet name="docs/docusaurus/docs/core/introduction/try_gx.py imports">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# <snippet name="docs/docusaurus/docs/core/introduction/try_gx.py set up">
+context = gx.get_context()
+batch = context.sources.pandas_default.read_csv(
```

Comment:
`sources` is now `data_sources` so this line should be:

```suggestion
batch = context.data_sources.pandas_default.read_csv(
```